esmvalcore/preprocessor/_time.py

```diff
@@ -177,6 +177,49 @@ def get_time_weights(cube):
     return time_weights
 
 
+def hourly_statistics(cube, hours, operator='mean'):
+    """
+    Compute hourly statistics.
+
+    Chunks time in x hours periods and computes statistics over them;
+
+    Parameters
+    ----------
+    cube: iris.cube.Cube
+        input cube.
+
+    hours: int
+        Number of hours per period. Must be a divisor of 24
+        (1, 2, 3, 4, 6, 8, 12)
```

Comment:
I suggest adding this list to the documentation for clarity.

Also 24 is a divisor of 24 😁
```suggestion
        (1, 2, 3, 4, 6, 8, 12, 24)
```